events/events.go

```diff
@@ -47,9 +47,11 @@ func (e *Events) Install(eng *engine.Engine) error {
 
 func (e *Events) Get(job *engine.Job) engine.Status {
 	var (
-		since   = job.GetenvInt64("since")
-		until   = job.GetenvInt64("until")
-		timeout = time.NewTimer(time.Unix(until, 0).Sub(time.Now()))
+		since      = job.GetenvInt64("since")
+		sinceValid = job.Getenv("since") != ""
```

Comment:
Actually I'm not fan of `var`. Can we just use if `job.Getenv("since") != ""` below?